doc/classes/ProjectSettings.xml

```diff
@@ -510,6 +510,9 @@
 		<member name="debug/gdscript/warnings/constant_used_as_function" type="int" setter="" getter="" default="1" deprecated="This warning is never produced. Instead, an error is generated if the expression type is known at compile time.">
 			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when a constant is used as a function.
 		</member>
+		<member name="debug/gdscript/warnings/deprecated_identifier" type="int" setter="" getter="" default="1">
+			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when deprecated keywords are used.
```

Comment:
```suggestion
			When set to [code]warn[/code] or [code]error[/code], produces a warning or an error respectively when deprecated identifiers are used. These include any deprecated class, enum, constant, signal, property, and method.
```

Reply:
Whoops, thanks for catching that typo! 😅 Listing out the possible kinds of deprecated items also sounds great.